_source/templates/booky/config.scss

```diff
@@ -26,7 +27,7 @@ $color--4: #eb65a0;
 $color--5: #9b539c;
 $color--6: #29aba4;
 $color--7: #657786;
-$color--8: #555;
+$color--8: $color--dark;
```

Comment:
Is there a deeper meaning to this numbers, is it self-explaining? Or can we give it a bit more love? ❤️

Reply:
They are colors that a user can choose, will rename them.